<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="sendFrameBox">
      <property name="title">
       <string>Send CAN frame</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="TraceMsgBox">
      <property name="title">
       <string>Trace Window</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_receiver">
          <attribute name="title">
           <string>Receiver</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTextEdit" name="textEdit_reseiveMsg"/>
           </item>
           <item>
            <widget class="QLabel" name="receiverCanBusStatusLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_sender">
          <attribute name="title">
           <string>Sender</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTextEdit" name="textEdit_sendMsg"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputValue"/>
       </item>
       <item>
        <widget class="QPushButton" name="computeButtion">
         <property name="text">
          <string>compute</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="resultlabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Calls</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionClear_log"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuCalls"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionClear_log"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="vectorcan.qrc">
     <normaloff>:/img/connect.png</normaloff>:/img/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="vectorcan.qrc">
     <normaloff>:/img/disconnect.png</normaloff>:/img/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionClear_log">
   <property name="icon">
    <iconset resource="vectorcan.qrc">
     <normaloff>:/img/clear.png</normaloff>:/img/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear log</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="vectorcan.qrc">
     <normaloff>:/img/application-exit.png</normaloff>:/img/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="vectorcan.qrc"/>
 </resources>
 <connections/>
</ui>
